middlewares/multer/avatarUpload.js

```diff
@@ -0,0 +1,35 @@
+const { ErrorObject } = require("../../helpers/error");
+const multer = require("multer");
+const ds = multer.diskStorage({
+  destination: (req, file, cb) => {
+    cb(null, "public/avatars");
+  },
+  filename: (req, file, cb) => {
+    req.body.avatar = `http://localhost:${process.env.PORT || 3000}/avatars/${
```

Comment:
this route is not implemented

Reply:
express.static(path.join(__dirname, "public"))
This is a route created and exposed by express.static, you can find it in app.js.